### PR TITLE
Fixes for SDL2_mixer

### PR DESCRIPTION
**Pull request by tpimh**
1. [This file](https://github.com/pspdev/psp-packages/blob/718fd247a1f2a21e29ce54358ff2271373b70523/sdl2-mixer/test.ogg.sample) won't play as it's HTML, fixed
2. Using `libmodplug` instead of `libxmp-lite` for MOD playback

### CODE REVIEW

**Comment by tpimh** [comment]
I only checked that the sample code compiles, but didn't test if it actually runs on PSP.

**Comment by tpimh** [comment]
Included example is a tiny bit slimmer when using modplug instead of xmp (from 4531372 bytes it went to 4399636 bytes). I was expecting it to be bigger because of libstdc++, but that's not the case. Can't compare against mikmod, because it's support was removed from SDL2_mixer.

File: sdl2-mixer/test.ogg.sample
Comment (sharkwouter) [inline sdl2-mixer/test.ogg.sample:1]: This file is required, but it seems it totally wrong. Can you replace it with this file: https://pspdev.github.io/resources/test.ogg

**Comment by sharkwouter** [comment]
Nevermind the comment, your solution makes sense to me. Thanks for fixing this.

**Pull request merged**